Source/ROLib/ModelDef/ROLModelModule.cs

```diff
@@ -427,25 +434,17 @@ public void UpdateEngineModuleThrust(ModuleEngines engine, float thrustScalePowe
         /// Update the input moduleRCS enabled, thrust and axis enable/disable status.  Calls rcs.OnStart() to update
         /// </summary>
         /// <param name="rcs"></param>
-        /// <param name="thrustScaleFactor"></param>
-        public void UpdateRCSModule(ModuleRCS rcs, float thrustScaleFactor)
+        public void UpdateRCSModule(ModuleRCS rcs)
         {
-            float power = 0;
             if (definition.rcsModuleData is ModelRCSModuleData data)
             {
-                power = data.rcsThrust;
-                float scale = Mathf.Sqrt(moduleHorizontalScale * moduleVerticalScale);
-                scale *= layout.modelScalarAverage();
-                power *= Mathf.Pow(scale, thrustScaleFactor);
                 rcs.enableX = data.enableX;
                 rcs.enableY = data.enableY;
                 rcs.enableZ = data.enableZ;
                 rcs.enablePitch = data.enablePitch;
                 rcs.enableYaw = data.enableYaw;
                 rcs.enableRoll = data.enableRoll;
             }
-            rcs.thrusterPower = power;
-            rcs.moduleIsEnabled = power > 0;
             rcs.OnStart(PartModule.StartState.Flying);
         }

```

Comment:
Actually, I believe this doesn't have any place in the model manipulation.  It may have been relevant for setting the power values.  But that's now properly handled elsewhere, and so should whatever fixup is necessary to the running state be.

Reply:
I'll remove the entire method here.  Setting those enabled fields should go into the ROE code to update the module using the data in the definition.